great_expectations/experimental/metric_repository/column_descriptive_metrics_metric_retriever.py

```diff
@@ -251,42 +269,57 @@ def _generate_column_metric_configurations(
 
     def _compute_metrics(
         self, batch_request: BatchRequest, metric_configs: list[MetricConfiguration]
-    ):
+    ) -> tuple[str, _MetricsDict, _AbortedMetricsInfoDict]:
         validator = self._context.get_validator(batch_request=batch_request)
         # The runtime configuration catch_exceptions is explicitly set to True to catch exceptions
         # that are thrown when computing metrics. This is so we can capture the error for later
         # surfacing, and not have the entire metric run fail so that other metrics will still be
         # computed.
-        # TODO: Get aborted_metrics in addition to computed metrics so they can be plumbed into the MetricException.
-        computed_metrics = validator.compute_metrics(
+        (
+            computed_metrics,
+            aborted_metrics,
+        ) = validator.compute_metrics_with_aborted_metrics(
```

Comment:
The naming of this method is a bit verbose, but as discussed, this is the lesser of two evils

Reply:
Thanks - hopefully I can refactor this method out soon by changing the callers of compute_metrics() to accept the aborted metrics as well.